react-app/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,35 +1,24 @@
-import React, { Component } from 'react';
+import React from 'react';
```

Comment:
`import React, {Fragment} from 'react'` and use directly Fragment instead of `React.Fragment`